.teamcity/src/vcs/Elasticsearch.kt

```diff
@@ -1,11 +1,13 @@
 package vcs
 
+import getCorrespondingESBranch
+import getProjectBranch
 import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
 
 object Elasticsearch : GitVcsRoot({
-  id("elasticsearch_master")
+  id("elasticsearch_${getProjectBranch()}")
```

Comment:
This id should match be unique to the upstream/name right?
```suggestion
  id("elasticsearch_${getCorrespondingESBranch()}")
```